### PR TITLE
Arvin react assessment 4 completed

### PR DESCRIPTION
**Pull request by adclleva**

### CODE REVIEW

File: react.md [hunk 1]
```diff
@@ -5,40 +5,77 @@ Answer the following questions. First, without external resources. Challenge you
...
+2 true? 
+3 true?
+4 true? 
+5 false
+6 true?
+7 true?
+
+- Researched Answers
+1 true 
+2 false?
```
Comment (sjproctor) [inline react.md:27]: Yup, this is false. React can be used with lots of different backend languages.

File: react.md [hunk 2]
```diff
@@ -5,40 +5,77 @@ Answer the following questions. First, without external resources. Challenge you
...
+3 true?
+4 true? 
+5 false
+6 true?
+7 true?
+
+- Researched Answers
+1 true 
+2 false? 
+3 true
```
Comment (sjproctor) [inline react.md:28]: This is false - React is only frontend. It doesn't do any server side work.

File: react.md [hunk 3]
```diff
@@ -5,40 +5,77 @@ Answer the following questions. First, without external resources. Challenge you
...
+4 true
+5 false
+6 true
+7 false
+
```
Comment (sjproctor) [inline react.md:33]: This is false - Technically JSX is rendered within a React component.

**Review by sjproctor — APPROVED** [review]
Nice work Arvin!

File: react.md (hunk 3 above)
Comment (adclleva) [reply react.md:33]: Thank you for the clarification!

**Pull request merged**